### PR TITLE
:sparkles: (signer-btc) [DSDK-475]: Implement GetExtendedPublicKey device action with DefaultSigner DI

### PR DESCRIPTION
**Pull request by jdabbech-ledger**
### 📝 Description

Implement SignerBtc, DI & GetExtendedPkeyDA

### ❓ Context

- **JIRA or GitHub link**: [DSDK-475]

- **Feature**:

### ✅ Checklist

Pull Requests must pass CI checks and undergo code review. Set the PR as Draft if it is not yet ready for review.

- [x] **Covered by automatic tests** 
- [x] **Changeset is provided** 
- [ ] **Impact of the changes:** 
  - _list of the changes_

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.

[DSDK-475]: https://ledgerhq.atlassian.net/browse/DSDK-475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/signer/signer-btc/src/api/SignerBtc.ts [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 export interface SignerBtc {
-  getExtendedPubkey: (
-    derivationPath: string,
-    checkOnDevice?: boolean,
-  ) => Promise<string>;
-  getAddress: (wallet: Wallet, options?: AddressOptions) => Promise<string>;
-  signMessage: (wallet: Wallet, message: string) => Promise<Signature>;
-  signPsbt: (wallet: Wallet, psbt: Psbt) => Promise<Psbt>;
-  signTransaction: (wallet: Wallet, psbt: Psbt) => Promise<Uint8Array>;
+  getExtendedPublicKey: (args: {
```
Comment (jiyuzhuang) [inline packages/signer/signer-btc/src/api/SignerBtc.ts:8]: [ASK] I think we should keep one parameter format for all three signers: `...args` or `{ ...args }`, I prefer the one used here.

File: packages/signer/signer-btc/src/internal/app-binder/di/appBinderTypes.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const appBinderTypes = {
+  AppBinder: Symbol.for("AppBinder"),
```
Comment (jiyuzhuang) [inline packages/signer/signer-btc/src/internal/app-binder/di/appBinderTypes.ts:2]: [ASK] The similar file in ETH signer should be updated: `AppBinding` => `AppBinder`, of course this will be done in another PR.

**Review by aussedatlo — APPROVED** [review]
